torch/nn/modules/module.py

```diff
@@ -583,8 +597,18 @@ def state_dict(self, destination=None, prefix='', keep_vars=False):
             ['bias', 'weight']
 
         """
-        if destination is None:
-            destination = OrderedDict()
+        if destination is not None:
+            raise ValueError("destination argument is deprecated")
+        if prefix is not '':
+            raise ValueError("prefix argument is deprecated")
```

Comment:
Oh that's not what I meant, sorry if I was unclear. :confused: I only wanted to remove the docs for those functions, no need to implement a private version of this method.

Reply:
I reverted this part in new commit. I did the deprecation because the `._metadata[prefix] = ...` thing will error if people actually used `destination` argument. But I think it should be fine as probably no one uses it.